docs/website/blog/2025-06-17-client-cli-cardano-database-backends.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Switching to Cardano database incremental certification
+authors:
+  - name: Mithril Team
+tags:
+  [
+    mithril client,
+    library,
+    cli,
+    Cardano database,
+    Cardano database v2,
+    certification,
+    backend,
+  ]
+---
+
+### Switching to Cardano database incremental certification
+
+We have recently introduced a new incremental certification process for the **Cardano node internal database**. This change is part of our ongoing efforts to enhance the Mithril protocol and improve the efficiency of Cardano node bootstrapping.
+
+More information about the Cardano database certifications is available at:
+
+- [Cardano node database](https://mithril.network/doc/mithril/advanced/mithril-certification/cardano-node-database)
+- [Cardano node database v2](https://mithril.network/doc/mithril/advanced/mithril-certification/cardano-node-database-v2)
+
+The **Cardano node database v2** certification is incremental and more efficient than the original **Cardano node database** certification. Thus it will progressively replace the current certification process for the Cardano node internal database.
```

Comment:
```suggestion
The Cardano node database v2 certification offers incremental verification and greater efficiency compared to the original method. It will progressively replace the current certification process for the Cardano node internal database.
```